File: java-store-management-system/src/Drink.java
import java.time.LocalDate;
import java.util.Date;

public class Drink extends Item {
    private Double volume;
    private boolean isAlcohol;

    public Drink() {
        super();
    }

    public Drink(Double volume, boolean isAlcohol) {
        super();
        this.volume = volume;
        this.isAlcohol = isAlcohol;
    }

    public Drink(String name, Double price, LocalDate expirationDate, Double volume, boolean isAlcohol) {
        super(name, price, expirationDate);
        this.volume = volume;
        this.isAlcohol = isAlcohol;
    }

    public Double getVolume() {
        return volume;
    }

    public void setVolume(Double volume) {
        this.volume = volume;
    }

    public boolean isAlcohol() {
        return isAlcohol;
    }

    public void setAlcohol(boolean alcohol) {
        isAlcohol = alcohol;
    }

    @Override
    public String toStr() {
        String message = super.toStr();
        message += String.format("\nVolume: %smL\nIs Alcohol: %s", volume, isAlcohol == true ? "Yes" : "No");
        return message;
    }
}

File: python-store-management-system/Drink.py
from Item import Item


class Drink(Item):
    def __init__(self, name, price, expiration_date, volume, is_alcohol):
        super().__init__(name, price, expiration_date)
        self.volume = volume
        self.is_alcohol = is_alcohol

    def __str__(self):
        return f"Item: {self.name}, {self.price}, {self.expiration_date}, {self.volume}, {self.is_alcohol}"
